resources/secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: objectstorage-provisioner
+  labels:
+    app.kubernetes.io/part-of: container-object-storage-interface
+    app.kubernetes.io/component: driver-ceph
+    app.kubernetes.io/version: main
+    app.kubernetes.io/name: cosi-driver-ceph
+type: Opaque
+#stringData:
+#  DriverPrefix: ""
```

Comment:
I'm having trouble understanding how the upper-camel-case format `DriverPrefix` here is related to configs elsewhere. Does this match up to the CLI flag 1-to-1?

Either way, K8s API standards generally have config keys being lower-camel-case like `driverPrefix`.